src/abstractions/software/assembler/code/index.js

```diff
@@ -10,7 +12,7 @@ const DESTIONATION_CODES = Object.freeze({
 })
 
 const JUMP_CODES = Object.freeze({
-  NULL: '000',
+  '': '000',
```

Comment:
Incorrect codes, should be handled by code

Reply:
I added a test for `code` module.